packages/mux-player-react/src/index.tsx

```diff
@@ -1,3 +1,5 @@
+'use client';
```

Comment:
could you double check to see if this sticks around after esbuild minifies?
I believe there was an issue with that, fix was to make this a esbuild banner in CLI args

Reply:
And, just in case you want to triple check, you'll find that the new [/app-router](https://elements-demo-nextjs-git-fork-decepulis-dc-rsc-mux.vercel.app/app-router) test page verifies that everything works as expected